src/index.ts

```diff
@@ -69,7 +69,7 @@ interface FilledOptions extends Options {
   minThreads : number,
   maxThreads : number,
   idleTimeout : number,
-  maxQueue : number | 'auto',
+  maxQueue : number,
```

Comment:
Is this intentional?

Reply:
Yes, because `maxQueue` is never `'auto'` there. In particular, this is relevant specifically when you’re using `maxQueue: 'auto'`, because then you have to check the value of `pool.options.maxQueue`, and now when you’re using TS you don’t have to explicitly assert that that is a number anymore.